### PR TITLE
style: better Drawer loading effect

### PR DESCRIPTION
**Pull request by afc163**
[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

- ✅ 优化 Drawer loading 的效果，改成包裹形式，这样会有半透明遮罩和渐变的效果。
- ⚠️ 缺点：会改变 Drawer panel 里的默认 dom 结构。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | Improve Drawer `loading` ui effect.          |
| 🇨🇳 Chinese | 优化 Drawer `loading` 加载效果。          |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/drawer/DrawerPanel.tsx [hunk 1]
```diff
@@ -156,23 +156,8 @@ const DrawerPanel: React.FC<DrawerPanelProps> = (props) => {
...
-          alignItems: 'center',
-        }}
-        {...spinProps}
-      />
-    );
-  }
-
   return (
-    <>
+    <Spin {...spinProps}>
```
Comment (zombieJ) [inline components/drawer/DrawerPanel.tsx:160]: Drawer 会和 Modal 对齐，改成 content skeleton。footer operation 进行 lock。这个可以先 hold 一下

Comment (afc163) [reply components/drawer/DrawerPanel.tsx:160]: content skeleton 我不同意，这个和 children 直接写 Skeleton 区别不大。

另外已经有内容的 Modal/Drawer 切 loading，高度会跳动。

Comment (li-jia-nan) [reply components/drawer/DrawerPanel.tsx:160]: master 分支我把行内样式改成了 css，可能会有冲突，要么这么先合一下？

- #48845

**Comment by li-jia-nan** [comment]
这样的话就没有垂直居中了：

image

**Pull request closed without merging**